reference/host-crd.md

```diff
@@ -1,119 +1,170 @@
 # The Host CRD
 
-The Ambassador Edge Stack automates the creation of TLS certificates via the [Edge Policy Console](../../about/edge-policy-console), which provides HTTPS for your hosts. Note that **in order to have TLS and automatic HTTPS, your host must be an FQDN as specified in the [product requirements](../../user-guide/product-requirements) page.**
+The custom Host resource defines how Ambassador will be visible to the outside world. It collects all the following information in a single configuration resource:
```

Comment:
This should really be "the `Host` custom resource" since "custom resource" and "custom resource definition" are Kube terms of art.

(I realize that I wrote that originally. That makes me sad. 🙁 )